operator/data/profiles/default.yaml

```diff
@@ -451,6 +451,7 @@ spec:
         type: ClusterIP
         env:
           ISTIO_META_ROUTER_MODE: "sni-dnat"
+        name: istio-egressgateway
```

Comment:
Is this addition necessary? just try to reduce the overlap of the settings in values and in components spec

Reply:
yes, it's required because i had to make changes in the charts to not hard code the gateway name. in the case where the name is not supplied in IOP, we need some default here.

Reply:
oh ok. Also I think the other issues complaining unable to set user defined gateways are caused by this as well?

Reply:
right